README.md

```diff
@@ -39,7 +39,7 @@ After opening the project in IntelliJ, double check that the Java SDK is properl
 
 Presto comes with sample configuration that should work out-of-the-box for development. Use the following options to create a run configuration:
 
-* Main Class: `io.prestosql.server.PrestoServer`
+* Main Class: `io.prestosql.server.Server`
```

Comment:
revert the rename, this is going to be annoyance for developers within the community (all the saved launch configurations...)

call the new class simply `Main`

Reply:
add some summary of changes (and purpose) to the `Run PrestoServer main on Java 8 for version check` commit, or split it up, as the purpose is hard to infer from the diff.

Reply:
No, keeping the class name is intentional as it shows up in `jps` and other JVM tools. Minimizing the administrator visible changes outweighs the minor developer inconvenience (it’s one change of the class name your IDE run configuration — we had a far larger change when we moved all the tests to a new package).

Reply:
Good idea, I added a commit description. The purpose is to fail with a friendly error message rather than *unsupported class file version*.